src/GalleryApp.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Header from './Components/Header';
+import Footer from './Components/Footer';
+
+class GalleryApp extends React.Component {
+
+  constructor(props) {
```

Comment:
You should deconstruct your props here so that you don't need to keep having props. later on

Reply:
Props deconstructed (i think this is the way) in constructor() and also added to set state

Reply:
Looks incorrect. Have a read of: https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0

Reply:
Reading the above article and merging present work